<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupConfigDialog</class>
 <widget class="QDialog" name="BackupConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>备份设置</string>
  </property>
  <widget class="QToolButton" name="browseButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择目录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cloudCheckBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>云备份（需要先登录账户）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="everyweekRadioButton_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>94</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>每周</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>28</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> 备份文件名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="backupFilenameLineEdit">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>28</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>94</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>   定时备份</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="passwordCheckBox_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>125</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>加密备份</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="noneRadioButton">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>94</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>不定时</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>123</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>61</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>     备份到</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="everydayRadioButton_2">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>94</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>每天</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="backupFileDirectoryLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>61</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
